static/app/views/eventsV2/savedQuery/index.tsx

```diff
@@ -92,9 +94,10 @@ class SavedQueryButtonGroup extends React.PureComponent<Props, State> {
 
     // For modifying a SavedQuery
     const isEqualQuery = nextEventView.isEqualTo(savedEventView);
+    const isEqualYAxis = isEqual(yAxis, savedQuery.yAxis);
```

Comment:
this is because we aren't using the yAxis from the event view and instead relying on the location params right?

Reply:
Yup! EventView only accepts single yAxis for now which makes it not reliable when checking for multi y axis changes so we need to check the full yAxis for any changes that may come from the location